code/modules/mob/living/carbon/human/species/station/diona/diona.dm

```diff
@@ -160,7 +161,7 @@
 /datum/species/diona/handle_death(var/mob/living/carbon/human/H, var/gibbed = 0)
 	if (!gibbed)
 		// This proc sleeps. Async it.
-		INVOKE_ASYNC(H, /mob/living/carbon/human/proc/diona_split_into_nymphs, TRUE)
+		INVOKE_ASYNC(H, /mob/living/carbon/human/proc/diona_split_into_nymphs)
```

Comment:
??? What the hell is going on here? Can you walk me through what this code does, because I seriously don't understand why you're lowering diona health but then later doing ` 	var/gestalt_health = (300 -  getFireLoss() - getBruteLoss() - getToxLoss()) / 6 `.

Reply:
I'm trying to make it so Dionae health = Nymph health * amount of the nymphs in the dionae. Dionae would have 300 health, but to give them a buffer so the nymphs don't instantly die when deathsplitting the Dionae's health is set to 240, the last 60 from the 300 is divided between the nymphs so they get 10 health no matter what. Originally they were just going to split once they reach 240 damage and have 300 health but this caused issues, so it was changed to this. The 300 health from var/gestalt_health = (300 - getFireLoss() - getBruteLoss() - getToxLoss()) / 6. is the Dionae's total health, plus it helps simplify the math.